README.md

```diff
@@ -42,6 +42,26 @@ $ cmake --build .
 This will build Fizzy as a library and since there is no public API
 (the so called *embedder API* in WebAssembly) yet, this is not very useful.
 
+## WASI
+
+Building with the `FIZZY_WASI` option will output a `fizzy-wasi` binary implementing
+the [WASI] API (a very limited subset of `wasi_snapshot_preview1`, to be precise).
```

Comment:
Should `wasi_unstable` be mentioned here?

Reply:
See above, will remove it.